test/test_type_hints.py

```diff
@@ -128,6 +128,7 @@ def test_doc_examples(self):
                 self.fail(f"mypy failed:\n{stdout}")
 
     @unittest.skipIf(not HAVE_MYPY, "need mypy")
+    @unittest.skipIf(os.getenv("IN_CI") is not None, "Already running in CI during lint")
```

Comment:
reran this locally on base commit shown 1282 file touched instead of 1281. wondering if we missed something.

Reply:
Thank you for pointing it out. Will do the comparison in the followup PR.